File: src/main/java/com/glass/java/act03/Act03.java
package com.glass.java.act03;

public class Act03 {

    public static void main(String[] args) {
    }

    public static void printAgeIfPerson(Object obj) {
        if (obj instanceof Person) {
            Person person = (Person) obj;
            System.out.println(person.getAge());
        }
    }
}

File: src/main/kotlin/com/glass/kotlin/act03/Act03.kt
package com.glass.kotlin.act03

import com.glass.java.act03.Person

fun main() {

    printAgeIfPerson(Person("KKK", 90))

    val name = "HIG"
    val str = """
        ABC
        EFG
        ${name}
    """.trimIndent()
}

fun printAgeIfPerson(obj: Any?) {
    val person = obj as? Person
    println(person?.age)
}